src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseUserServiceImpl.java

```diff
@@ -728,6 +732,8 @@ public U getByUuidAndInitializeCollections(String uuid) {
         if (!isEmpty(tenantIds)) {
             spec = spec.and(usersIn(tenantIds));
         }
+
+        spec = spec.and(fieldIsNull("deleted"));
```

Comment:
The column type is the TIMESPAMP - it will not be possible to set FALSE
There are existing column created/updated so "deleted" follows the same naming convention